development/ts-migration-dashboard/common/build-module-partitions.ts

```diff
@@ -328,8 +358,20 @@ function partitionModulesByLevel(
   }
   Object.values(modulesById).forEach((module) => {
     modulePartitions[module.level].children.push({
-      name: module.id,
+      id: module.id,
       hasBeenConverted: module.hasBeenConverted,
+      dependentIds: Object.keys(module.dependents).sort(),
+      dependencyIds: Object.keys(module.dependencies).sort(),
+    });
+  });
+  Object.values(modulePartitions).forEach((partition) => {
```

Comment:
This change is not strictly necessary to this PR, but it was a bit annoying that the boxes in each level were not sorted by filename, so this change does that.